lib/src/mapbox_map.dart

```diff
@@ -219,6 +220,12 @@ class MaplibreMap extends StatefulWidget {
   /// Override hybrid mode per map instance
   final bool? useHybridCompositionOverride;
 
+  /// This will show an info button at the bottom right of the map that
+  /// shows information about the map sdk.
+  /// This value is only set initially and changing this after the initial creation
+  /// will have no effect.
```

Comment:
```suggestion
  /// This will show an info button at the bottom right of the map that
  /// shows information about the map sdk. 
  /// It can also show information about the sources of the displayed map data 
  /// if the used style JSON includes the relevant attribution property.
  /// (See https://maplibre.org/maplibre-style-spec/sources/#vector-attribution)
  ///
  /// Important: please make sure to follow the licenses of the software libraries 
  /// and data sources you use, by providing attribution where necessary.
  ///
  /// This value is only set initially and changing this after the initial creation
  /// will have no effect.
```

Reply:
LGTM in general, I'd just like to slightly extend the documentation here, because hiding this information button might mean that legally required attribution is no longer shown, so I think developers should think about that. What do you think?

Reply:
This is a very good point!
I will apply your suggestions when I touch the branch the next time :)